app/assets/stylesheets/modules/_selects.styl

```diff
@@ -34,5 +34,6 @@
     right 191px
   select.sorts
     background url('../images/sort.svg') no-repeat 48% white
+    color: transparent
```

Comment:
Is this the cause of the white-on-white select menu?

![white on white](https://cloud.githubusercontent.com/assets/848483/21432589/0e52123e-c821-11e6-910c-b487edd546eb.png)